### ISSUE TITLE
Changing the size of partially visible items triggers list to scroll 

### ISSUE THREAD

**Issue by Karpengold** [tannerlinsley/react-virtual#188]
According to this line https://github.com/tannerlinsley/react-virtual/blob/433d9663cedb7332d738a236199f9f7e76639039/src/index.js#L178

in case of changing the size of partially visible items (for example expanding) list is scrolling to save same position for partially visible item: https://codesandbox.io/s/react-virtual-resizeobserver-transition-forked-65sg7?file=/src/App.js

https://user-images.githubusercontent.com/17138959/129550951-60a11154-48a6-42cd-a9d6-ccf818ad9bef.mov

**Comment by piecyk** [comment]
Adjusting scroll position when item size change should only happen when we are scrolling backward. Looks like this should fix the this issue https://github.com/tannerlinsley/react-virtual/compare/master...piecyk:fix/adjust-scroll-position-when-item-size-change-only-when-scrolling-backward

**Comment by Karpengold** [comment]
@piecyk cool! As I understood this feature also provide ability to get isScrolling param from react-virtual hook, so I will be able to remove custom scroll handling logic :)

**Comment by piecyk** [comment]
@Karpengold yes, as we want to reset scrollDirection, then having extra isScrolling looked like good idea 🤔 

Let's see what @tannerlinsley will say.

**Comment by Karpengold** [comment]
@piecyk  I found the little hack how to prevent this behaviour 
I use paddingStart property and add some extra space in the beginning (i.e. 1 item height)
And during calculating the start property of each item in the list I decrease it by this extra space

**Comment by tannerlinsley** [comment]
This is a decent workaround for now. We'll work on improving this experience in the next major version for sure.
